src/pages/apps/[id]/settings/_actions/outbound_webhook_actions.tsx

```diff
@@ -2,6 +2,7 @@ import { useEffect, useState } from "react";
 import { useLocation } from "react-router-dom";
 import { RootContextProps } from "~/pages/Root.context";
 import type { LocationState, OutboundWebhook } from "../types";
+import _ from "lodash";
```

Comment:
As far as I remember, this will import the whole library and will increase the page size. We can simply import the function we need to reduce the page size.